### PR TITLE
Maintenance Week: add tests for ZooniverseLogo, ZooniverseLogoType

### PR DESCRIPTION
**Pull request by shaunanoordin**
## PR Overview

Package: `lib-react-components`
Part of: Maintenance Week Oct 2022

Adds React Testing Library tests for the ZooniverseLogo and ZooniverseLogotype components

### CODE REVIEW

**Comment by shaunanoordin** [comment]
OK, this is ready for review!

**Comment by eatyourgreens** [comment]
I’ve removed these tests in #4732. We should probably coordinate better!

File: packages/lib-react-components/src/ZooniverseLogo/ZooniverseLogo.spec.js [hunk 1]
```diff
@@ -8,37 +8,46 @@ describe('ZooniverseLogo', function () {
...
+        <ZooniverseLogo id={ID} size={SIZE} />
+      )
+    })
+
+    it('should render without crashing', function () {
+      expect(screen).to.be.ok()
+    })
+
+    it('should use the `id` prop for `aria-labelledby` and `title`', function () {
+      const zooLogo = screen.getByLabelText('Zooniverse Logo')  // Gets the SVG
```
Comment (eatyourgreens) [inline packages/lib-react-components/src/ZooniverseLogo/ZooniverseLogo.spec.js:23]: You don’t need this test at all.

File: packages/lib-react-components/src/ZooniverseLogo/ZooniverseLogo.spec.js [hunk 2]
```diff
@@ -8,37 +8,46 @@ describe('ZooniverseLogo', function () {
...
-        id={ID}
-        size={SIZE}
-        foo={FOO}
-      />
-    )
-
-    expect(wrapperWithProps.find('svg').prop('foo')).to.equal(FOO)
+  describe('with additional props', function () {
+    it('should pass through any other props to the SVG', function () {
+      const FOO = 'bar'
```
Comment (eatyourgreens) [inline packages/lib-react-components/src/ZooniverseLogo/ZooniverseLogo.spec.js:39]: Maybe use a valid SVG attribute here.

File: packages/lib-react-components/src/ZooniverseLogotype/ZooniverseLogotype.spec.js [hunk 3]
```diff
@@ -8,41 +8,50 @@ describe('ZooniverseLogotype', function () {
...
+        <ZooniverseLogotype id={ID} width={WIDTH} />
+      )
+    })
+
+    it('should render without crashing', function () {
+      expect(screen).to.be.ok()
+    })
+
+    it('should use the `id` prop for `aria-labelledby` and `title`', function () {
+      const zooLogo = screen.getByLabelText('Zooniverse Logo')  // Gets the SVG
```
Comment (eatyourgreens) [inline packages/lib-react-components/src/ZooniverseLogotype/ZooniverseLogotype.spec.js:23]: This test is redundant too.

File: packages/lib-react-components/src/ZooniverseLogotype/ZooniverseLogotype.spec.js [hunk 4]
```diff
@@ -8,41 +8,50 @@ describe('ZooniverseLogotype', function () {
...
+    })
+
+    it('should set the width from the `width` prop', function () {
+      const zooLogo = screen.getByLabelText('Zooniverse Logo')  // Gets the SVG
+      expect(parseInt(zooLogo.getAttribute('width'))).to.equal(WIDTH)
+    })
+
+    it('should calculate the height', function () {
+      const zooLogo = screen.getByLabelText('Zooniverse Logo')  // Gets the SVG
+      expect(zooLogo.getAttribute('height')).to.be.ok()
```
Comment (eatyourgreens) [inline packages/lib-react-components/src/ZooniverseLogotype/ZooniverseLogotype.spec.js:37]: Can you check the expected height in pixels here?

File: packages/lib-react-components/src/ZooniverseLogotype/ZooniverseLogotype.spec.js [hunk 5]
```diff
@@ -8,41 +8,50 @@ describe('ZooniverseLogotype', function () {
...
-        id={ID}
-        width={WIDTH}
-        foo={FOO}
-      />
-    )
-
-    expect(wrapperWithProps.find('svg').prop('foo')).to.equal(FOO)
+  describe('with additional props', function () {
+    it('should pass through any other props to the SVG', function () {
+      const FOO = 'bar'
```
Comment (eatyourgreens) [inline packages/lib-react-components/src/ZooniverseLogotype/ZooniverseLogotype.spec.js:43]: Again, better to use a valid SVG attribute here.

File: packages/lib-react-components/src/ZooniverseLogotype/ZooniverseLogotype.spec.js [hunk 6]
```diff
@@ -8,41 +8,50 @@ describe('ZooniverseLogotype', function () {
...
+
+      render(
+        <ZooniverseLogotype
+          id={ID}
+          width={WIDTH}
+          foo={FOO}
+        />
+      )
+
+      const zooLogo = screen.getByLabelText('Zooniverse Logo')  // Gets the SVG
```
Comment (eatyourgreens) [inline packages/lib-react-components/src/ZooniverseLogotype/ZooniverseLogotype.spec.js:53]: I’ve changed this label to ‘Zooniverse’ in #3732.

**Comment by shaunanoordin** [comment]
Let's get #3732 merged first, then I'll rebase and make the necessary changes.

**Comment by eatyourgreens** [comment]
I wrote tests for the logotype in #2732, but it might be good to duplicate those tests here. I’m not sure. If not, I think the logotype should be moved to `ZooFooter/components` as it’s only used in the footer, and is already tested there.

**Pull request closed without merging**